docs: add inventory of assets belonging to AsyncAPI Initiative (#2024)
asyncapi/community: docs/020-governance-and-policies/initiative-inventory.md

---
title: List Of People and Assets AsyncAPI Depends On
weight: 25
---

## Payment/Donations

- Open Collective and multiple sub-projects: https://opencollective.com/asyncapi
- Linux Foundation Crowdfunding: https://crowdfunding.lfx.linuxfoundation.org/projects/asyncapi
- GitHub Sponsors: https://github.com/sponsors/asyncapi

## Domains

Managed by Linux Foundation

- asyncapi.com
- asyncapi.org

## Email

- Google Workspace:
  - owned by Fran

>   - connected domain: asyncapi.io

Reviewer: ```suggestion
  - connected domains: asyncapi.com and asyncapi.io (just there temporarily so we don't lose any emails). Each email below has its corresponding `@asyncapi.io` alias too.
```

  - used email addresses:

>     - info@asyncapi.io (used for contact with partners/sponsors and for security reports)
>     - press@asyncapi.io (not sure what the status is, never accessed it, old email)
>     - bot@asyncapi.io and alias bot+eve@asyncapi.io are used for two AsyncAPI GitHub bot accounts

Reviewer: ```suggestion
    - info@asyncapi.com (used for contact with partners/sponsors and for security reports)
    - press@asyncapi.com (not sure what the status is, never accessed it, old email)
    - bot@asyncapi.com and alias bot+eve@asyncapi.com are used for two AsyncAPI GitHub bot accounts
```

For plugins:
- [Visual Studio Marketplace](https://marketplace.visualstudio.com/publishers/asyncapi): We have an account under which we publish https://github.com/asyncapi/vs-asyncapi-preview automatically
- [Open VSX Registry](https://open-vsx.org/namespace/asyncapi): We have an official account to publish https://github.com/asyncapi/vs-asyncapi-preview in automated way
- [JetBrains](https://plugins.jetbrains.com/plugin/15673-asyncapi#): No official account to publish https://github.com/asyncapi/jasyncapi-idea-plugin; published manually. Currently, https://github.com/Pakisan takes care of it.

## Amazing Humans

People we hire through Open Collective contracts or pay for a specific job to be done in one of our official programs. All costs below are always planned in the budget and approved by TSC. For clarity and better budget planning goal is to have dedicated subprojects in Open Collective platform to secure funding more easily.

- [Thulie](https://github.com/thulieblack): hired through official contract with Open Source Collective (on our behalf) to do community management ($2.5k monthly). Managed through https://opencollective.com/asyncapi/projects/community-manager-budget

> - [Ash](https://github.com/aeworxet): makes sure AsyncAPI Bounty program runs smoothly, organizes 4 rounds each quarter. We pay $400 for each round. We started to manage funding through https://opencollective.com/asyncapi/projects/asyncapi-bounty-program

Reviewer: I guess you meant this instead?

```suggestion
- [Ash](https://github.com/aeworxet): makes sure AsyncAPI Bounty program runs smoothly, organizes 4 rounds/year, one each quarter. We pay $400 for each round. We started to manage funding through https://opencollective.com/asyncapi/projects/asyncapi-bounty-program
```

> - [Bami](https://github.com/iambami): hired through official contract with Open Source Collective (on our behalf) to do community marketing ($2k monthly). Managed through https://opencollective.com/asyncapi/projects/community-marketing-specialist

Reviewer: Let's remove Bami from this list as this is no longer true.

```suggestion
```

- [Ace](https://github.com/AceTheCreator): ensures we participate in different mentorship programs and also host our own. We pay $2k for each program ($1k for all the work to kick off the program, get funding, and communications; and $1k at the end once completed and the summary is published). There is no single Open Collective subproject for securing funds. Until now, we have created dedicated projects per mentorship, but with the introduction of [cohorts](https://www.asyncapi.com/blog/2024-mentorship-program-summary#refocus-setting-our-future-intentions), it may make sense to maintain one project for all mentorships, with funds secured for Ace’s payment too.

## Social Media

- Buffer: we pay for it $480 once a year. Used to post in one place and send posts to all social media
- LinkedIn: https://www.linkedin.com/company/asyncapi
- X: https://x.com/AsyncAPISpec
- Mastodon: https://fosstodon.org/@AsyncAPISpec
- Twitch: https://www.twitch.tv/asyncapi

## Messengers

- Slack: Pro Plan for free. We don't need to renew anything.

## Meetings

- [Restream](https://restream.io/): Streaming to different social media channels. Free pro access. Needs renewal every year.

> - Zoom: Paid service. We approve it as part of the budget once a year.

Reviewer: ```suggestion
- Zoom: Paid service. We approve it as part of the budget once a year. Paid by Fran (he expenses it through Open Collective after paying).
```

---
title: List Of People and Assets AsyncAPI Depends On
weight: 25
---

## Payment/Donations

- Open Collective and multiple sub-projects: https://opencollective.com/asyncapi
- Linux Foundation Crowdfunding: https://crowdfunding.lfx.linuxfoundation.org/projects/asyncapi
- GitHub Sponsors: https://github.com/sponsors/asyncapi

## Domains

Managed by Linux Foundation

- asyncapi.com
- asyncapi.org

## Email

- Google Workspace:
  - paid by Fran (he expenses it through Open Collective after paying)
  - connected domains: asyncapi.com and asyncapi.io (just there temporarily so we don't lose any emails). Each email below has its corresponding `@asyncapi.io` alias too.
  - used email addresses:

>     - info@asyncapi.com (used for contact with partners/sponsors and for security reports)
>     - press@asyncapi.com (not sure what the status is, never accessed it, old email)
>     - bot@asyncapi.com and alias bot+eve@asyncapi.com are used for two AsyncAPI GitHub bot accounts

Reviewer: ```suggestion
    - info@asyncapi.com (used for contact with partners/sponsors and for security reports)
    - press@asyncapi.com (not monitored, references have been removed so we can delete this email in a few months)
    - bot@asyncapi.com and alias bot+eve@asyncapi.com are used for two AsyncAPI GitHub bot accounts
    - conduct@asyncapi.com (used for getting in touch with the Code of Conduct Committee)
    - security@asyncapi.com (used for reporting security issues)
    - thulie@asyncapi.com (personal email for Thulie)
    - hugo@asyncapi.com (personal email for Hugo)
```
